practice-1/calculate_age.py

```diff
@@ -0,0 +1,10 @@
+def calculate_age(number):
+    age_reverse = list(str(number))
+    reversed_number = int(''.join(age_reverse[::-1]))
+    if int(number) + 36 == reversed_number:
+        print(number)
```

Comment:
There should be blank lines between functions, script blocks, functions and function calls, variables, constants ...
Please check for all.